pkg/updater/resultstore/resultstore.go

```diff
@@ -458,6 +525,12 @@ func processGroup(tg *configpb.TestGroup, group *invocationGroup) *updater.Infla
 						headers[i] = append(headers[i], targetHeaders...)
 					}
 				}
+
+				properties := map[string][]string{}
+				testSuite := sar.ActionProto.GetTestAction().GetTestSuite()
+				for _, t := range testSuite.GetTests() {
+					appendProperties(properties, t, nil)
```

Comment:
Hmm, it looks like we collect target result properties as well, but not for this (just for cell.Metrics instead; see targetResultNumerics() internally). I think it's worth a TODO to check if we need to include the properties from the target in addition to its test cases.